TBot/Program.cs

```diff
@@ -1575,6 +1597,18 @@ class Program {
 
 									// Add each planet that has inactive status to farmTargets.
 									foreach (Celestial planet in scannedTargets) {
+										if (Helpers.IsSettingSet(settings.AutoFarm.MinimumPlayerRank) &&
+											settings.AutoFarm.MinimumPlayerRank != 0 && (int) settings.AutoFarm.MinimumPlayerRank < ((Planet) planet).Player.Rank) {
```

Comment:
- [ ] Fix: Cannot convert Moon to Planet - Moon does not have Rank information, get parent Planet and retrieve Rank.

Reply:
a probaly not too heavy solution could be storing /api/players.xml and /api/universe.xml, look for the moon id in universe.xml, getting the player id and then check points/rank in players.xml
players.xml is updated once a day, so it could be safely cached on something like sleep mode's wakeup and not be called too often
universe.xml is updated once a week though, it may (and will) miss many celestials so i would keep a fallback (ignore rank is just fine as a fallback)